_episodes/03-github-pages.md

```diff
@@ -1,21 +1,160 @@
+   
+    Contact us at [mailto: {{ "{{ site.email " }}}}]({{ "{{ site.email " }}}}).
+   ~~~     
+   {: .language-md}
+ 
+2. Note that site parameters will not render nicely in GitHub (they will be displayed as text 
+{{ "{{ site.PARAMETER_NAME " }}}} rather than the parameter's rendered value) but will in the website.
+
+
+
+> ## Site-wide and page parameters 
+>
+> In addition to site-wide configuration available via the `site` global variable, page-specific information is 
+> available to you via the `page` global variable. Some of these are pre-defined (like `page.title`); 
+> others you can define yourself. Check this [list of predefined page parameters](https://jekyllrb.com/docs/variables#page-variables).
+>
+{: .callout}
+
+## More Complex Parameters
+
+We can define more complex parameters such as list and dictionaries (hashes) too. 
+For example, we may want 
+to define the names of the project team members, their roles and start dates inside the `_config.yml` file 
+and make use of this information throughout the website. To do so, in YAML notation, would look something like:
+
+~~~
+team_members: [
+    {name: "Jane Smith", role: "maintainer", start_date: "2018-03-15"},
+    {name: "Albert Hamilton", role: "editor", start_date: "2017-12-01"},
+    {name: "Alice Dauncey", role: "developer", start_date: "2020-01-04"}
+]
+~~~   
+{: .language-yaml}
+This defines team_members as a list of 3 elements; each element is a hash with key-value pairs for name, role and date. 
+
+> ## Indentation in YAML
+> Note that indentation level in YAML is important - it can be a cause of some not-so-obvious mistakes. Use online YAML
+>code validators, such as [YAML Lint](http://www.yamllint.com/), to make sure your YAML is correctly formatted.
+>
+{: .callout}
+
+We can now add this information to our website.
+
+1. Modify `_config.yml` file and add the `team_members` parameter as defined above. The file should now look like: 
+
+    ~~~
+    title: "Research Project Website"
+    description: "Research Project Description"
+    email: team@my.research.org 
+    team_members: [
+        {name: "Jane Smith", role: "maintainer", start_date: "2018-03-15"},
+        {name: "Albert Hamilton", role: "editor", start_date: "2017-12-01"},
+        {name: "Alice Dauncey", role: "developer", start_date: "2020-01-04"}
+        ]
+    ~~~   
+    {: .language-yaml}
+
+2. In file `team.md`, iterate over the values defined in parameter `site.team_members` in a loop to display all 
+team members' names and roles:
+
+    ~~~                            
+    ## Project team members  
+    The following people are members of our research team:
+    {{ "{% for team_member in site.team_members " }}%} 
+        - Name: {{ "{{ team_member.name " }}}} , role: {{ "{{ team_member.role " }}}}   
+    {{ "{% end_for " }}%}
+    ~~~   
+    {: .language-md}
+    TODO: definitely an overkill putting Liquid code here - this should go into the next episode. Which makes me
```

Comment:
Perhaps this can be replaced with a demonstration of how an individual value could be accessed from a structure like this, i.e. `site.team_members[0]["role"]`

Reply:
not necessary to do this now though: just making a note for later...

Reply:
I had that same idea in order not to use loops at this point @tobyhodges ! Perhaps to access the team-lead who is the first in the list? A bi contrived example but hey.

Reply:
Fixed in d557bb4.